src/sessions/session-manager.ts

```diff
@@ -253,6 +253,11 @@ export class SessionManager {
     return Promise.all(promises);
   }
 
+  async updateSessionWithDefaultResolution (track: MediaStreamTrack, session: IExtendedMediaSession): Promise<void> {
```

Comment:
NIT: this name could probably be more generic. I know you are only using it for the resolution stuff, but the guts of the function don't really do anything with resolution. It is really just adding or replacing a track on the session. Maybe naming it something like `addOrReplaceTrackOnSession()` would be better? Then it would be more clear what it is doing.